lib/gtfs_realtime_viz/state.ex

```diff
@@ -14,21 +14,67 @@ defmodule GTFSRealtimeViz.State do
 
   @spec init(:ok) :: {:ok, state}
   def init(:ok) do
-    {:ok, %{}}
+    {:ok, %{vehicles: %{}, trip_updates: %{}}}
```

Comment:
Should this be `%{vehicles: [], trip_updates: []}`? Since we're storing lists in here, I think our init should have an empty list somewhere.

Reply:
ah, true.

Reply:
actually, these are correctly maps - they still maintain the map of for example: `%{vehicles: %{prod: [], test: []}`

Reply:
should we add the specific environments we expect?  that seems a little presumptuous.

Reply:
Ahhh, right. No, that's fine.